Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:6.12.0
```

Comment:
Don't we want to keep using `6.11.2-alpine` base image (alpine ones are usually smaller)?

Reply:
we can, but we'd need to go through extra setup to get make etc. dependencies of node-gyp & friends working